smaupin/week03/w03d05/Micro-blog-with-Mongo

```diff
@@ -0,0 +1 @@
+Subproject commit e8e8bfb04b4d7fdd09791551b2c64326009fb032
```

Comment:
sure thing. I dont know what it is or how to do it, but i am happy to
correct my error whenever.

On Mon, Nov 9, 2015 at 10:20 AM, tgaff <notifications@github.com> wrote:

> In smaupin/week03/w03d05/Micro-blog-with-Mongo
> <https://github.com/sf-wdi-22-23/23-homework/pull/130#discussion_r44309371>
> :
>
> > @@ -0,0 +1 @@
> > +Subproject commit e8e8bfb04b4d7fdd09791551b2c64326009fb032
>
> Hey @smaupin <https://github.com/smaupin> This looks like you have an
> embedded repo here. Can we fix this sometime soon? I can't see the files in
> this directory at all. Let me know when you have a few minutes. [image:
> :camel:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sf-wdi-22-23/23-homework/pull/130/files#r44309371>.
>